### ISSUE TITLE
a new rule is needed: CREATE PROCEDURE

### ISSUE THREAD

**Issue by antonov-impulsm** [polymorphm/pg-dump-splitter#5]

**Comment by polymorphm** [comment]
the examples of SQL:

handmade:

```
create schema test_proc_schema
        authorization my_role;

create or replace procedure test_proc_schema.my_proc (
        _who text
    )
    language plpgsql
as $body$begin
    raise notice $function$hello % BEFORE commit$function$, _who;
    commit;
    raise notice $procedure$hello % AFTER commit$procedure$, _who;
end$body$;

alter procedure test_proc_schema.my_proc (
        _who text
    )
    owner to my_role;
```

pg_dump:

```
--
-- PostgreSQL database dump
--

-- Dumped from database version 11.2
-- Dumped by pg_dump version 11.2

SET statement_timeout = 0;
SET lock_timeout = 0;
SET idle_in_transaction_session_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SELECT pg_catalog.set_config('search_path', '', false);
SET check_function_bodies = false;
SET client_min_messages = warning;
SET row_security = off;

--
-- Name: test_proc_schema; Type: SCHEMA; Schema: -; Owner: my_role
--

CREATE SCHEMA test_proc_schema;


ALTER SCHEMA test_proc_schema OWNER TO my_role;

--
-- Name: my_proc(text); Type: PROCEDURE; Schema: test_proc_schema; Owner: my_role
--

CREATE PROCEDURE test_proc_schema.my_proc(_who text)
    LANGUAGE plpgsql
    AS $_$begin
    raise notice $function$hello % BEFORE commit$function$, _who;
    commit;
    raise notice $procedure$hello % AFTER commit$procedure$, _who;
end$_$;


ALTER PROCEDURE test_proc_schema.my_proc(_who text) OWNER TO my_role;

--
-- PostgreSQL database dump complete
--
```

**Comment by polymorphm** [comment]
fixed in 0311cd8
